.github/workflows/conformance-externalworkloads-v1.10.yml

```
@@ -72,8 +72,8 @@ jobs:
     name: Deduce required tests from code changes
     if: |
       (github.event_name == 'issue_comment' && (
-        (startsWith(github.event.comment.body, 'ci-external-workloads-v1.10') ||
-        startsWith(github.event.comment.body, 'test-backport-1.10')) &&
+		startsWith(github.event.comment.body, 'ci-external-workloads-v1.10') ||
+        (startsWith(github.event.comment.body, 'test-backport-1.10') && (needs.check_changes.outputs.tested == 'true'))
```

Think the `(needs.check_changes.outputs.tested == 'true')` incorrectly sneaked in there:
```suggestion
        startsWith(github.event.comment.body, 'ci-external-workloads-v1.10') ||
        (startsWith(github.event.comment.body, 'test-backport-1.10'))
```

---

I copy-pasted the https://github.com/cilium/cilium/pull/17019#pullrequestreview-719096795 patch. Let me remove it.